### DISCUSSION TITLE
4.36 - 泛函分析基础：学习指导和习题详解

### DISCUSSION THREAD

**Discussion by giscus** [duskmoon314/THU_BFA#12] [Announcements]
# 4.36 - 泛函分析基础：学习指导和习题详解

https://duskmoon314.github.io/THU_BFA/ch4/ex36.html

**Comment by SuBonan** [comment]
看起来没有用到“X是Banach空间”这个条件？

**Reply by duskmoon314** [reply]
说实话，我已经不太记得了 orz 似乎典范映射是对 X 有要求来着……

这个当时是对照着参考解答修正了一下表述，如果还不太对，欢迎帮忙修改～

**Reply by SuBonan** [reply]
我今天上课问了一下步老师，这里的确不需要要求X是Banach空间

**Comment by jefflyy** [comment]
看起来 $M$ 和 $\theta_n$ 有关？那这个 $M$ 不能用来控制待证的对任意 $f$ 都成立的不等式吧......

还是说有某种方法能说明 $\sup\limits_{\theta}M(\theta)\lt\infty$ 吗？

**Reply by SuBonan** [reply]
根据一致有界性原理，一族泛函g_{N, \theta}是一致有界的，所以不依赖于\theta。

注意N，\theta都是指标集，所以一致有界性就保证了

**Reply by jefflyy** [reply]
哇比作者还快😭
